### PR TITLE
Matthewcurtis1999/vir 1872 Syrupy Update

### PR DESCRIPTION
**Pull request by mattcurts**
Updated Syrupy and refreshed all snapshots that needed refreshing

### CODE REVIEW

**Comment by mattcurts** [comment]
currently blocked by subtractions/test_tasks local snapshot not matching with the snapshot generated during CI.

File: tests/uploads/__snapshots__/test_api.ambr [hunk 1]
```diff
@@ -339,82 +340,22 @@
...
+        }),
+      }),
+    ]),
     'page': 1,
     'page_count': 1,
     'per_page': 25,
     'total_count': 3,
-  }
----
-# name: TestFind.test_uploads_order
```
[the remark this answers is not in the document]
Comment (mattcurts) [reply tests/uploads/__snapshots__/test_api.ambr:355]: yes, there is no accompanying test to go with the snapshot. Did you want a test added?

File: tests/uploads/__snapshots__/test_api.ambr [hunk 2]
```diff
@@ -425,122 +366,42 @@
...
-    'user': <class 'dict'> {
+    'user': dict({
       'administrator': False,
       'handle': 'bob',
       'id': 'fb085f7f',
-    },
-  }
----
-# name: TestUpload.test[hmm]
-  <class 'dict'> {
```
[the remark this answers is not in the document]
Comment (mattcurts) [reply tests/uploads/__snapshots__/test_api.ambr:436]: yes, there is no accompanying test to go with the snapshot. Did you want a test added?

File: tests/uploads/__snapshots__/test_api.ambr (hunk 1 above)
[the remark this answers is not in the document]
Comment (igboyes) [reply tests/uploads/__snapshots__/test_api.ambr:355]: No. Just double-checking.

File: tests/uploads/__snapshots__/test_api.ambr (hunk 2 above)
[the remark this answers is not in the document]
Comment (igboyes) [reply tests/uploads/__snapshots__/test_api.ambr:436]: No. Just double-checking.

**Pull request merged**